src/fidesctl/core/system.py

```diff
@@ -234,9 +245,12 @@ def generate_system_aws(
     filtered_aws_systems = filter_aws_systems(
         systems=aws_systems, organization=organization
     )
```

Comment:
we should include the filtering code in the shared function. is there a reason you kept it separate?

Reply:
I think it should still be in there and included as part of #642 
I ended up bypassing the `generate_system_aws()` function and using the shared parts explicitly (thinking we might do similar in the future with moving these to the cli command) - [should be here](https://github.com/ethyca/fides/pull/642/files#diff-31b8e25ead58548d828f1a5bbb18814c881f53fcdee25e2f3a2c97dba8b3df12R84)

Reply:
Made the change we chatted about @earmenda if you have time to take another look and confirm 👍🏽